playwright-tests/playwright.config.ts

```diff
@@ -0,0 +1,44 @@
+// *****************************************************************************
+// Copyright (C) 2021-2023 logi.cals GmbH, EclipseSource and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0.
+//
+// This Source Code may also be made available under the following Secondary
+// Licenses when the conditions for such availability set forth in the Eclipse
+// Public License v. 2.0 are satisfied: GNU General Public License, version 2
+// with the GNU Classpath Exception which is available at
+// https://www.gnu.org/software/classpath/license.html.
+//
+// SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
```

Comment:
Do you need the classpath exception? The original file that you based it on had it but this folder has only license EPL-2.0. Also, can config files be copyrighted?

Reply:
I'm probably repeating myself, but I think we should not modify the original copyright header. However, in the context of this repository, its effective license is indeed EPL-2.0 (as would be reflected if we decided to add a second copyright header for our eventual modifications in this file).

Reply:
@marcdumais-work  could you please reply to this one?